islandora-connector-houdini/src/test/resources/AS2Event.jsonld

```diff
@@ -14,26 +14,28 @@
       ]
    },
    "object":{
-      "id":"urn:uuid:3d3550ff-1d4c-4484-a406-8f97c1673e5c",
+      "id":"urn:uuid:72358916-51e9-4712-b756-4b0404c91b1d",
       "url":[
          {
             "name":"Canoncial",
```

Comment:
This should be "Canonical" instead of Canocial

Reply:
Good catch @DiegoPino.  Thanks man.  I'll update that shortly.

Reply:
LOL, found the source of it: https://github.com/Islandora-CLAW/islandora/blob/8.x-1.x/src/EventGenerator/EventGenerator.php#L114

Reply:
cool!